tools/docs/github-query-items.ts

```diff
@@ -46,6 +29,34 @@ export interface Items {
   features: ItemsEntity[];
 }
 
+const GhOutputSchema = z.array(
+  z.object({
+    url: z.string(),
+    title: z.string(),
+    labels: z.array(
+      z.object({
+        name: z.string(),
+      }),
+    ),
+    number: z.number(),
+  }),
+);
+
+async function getIssuesByIssueType(
+  issueType: 'Bug' | 'Feature',
+): Promise<ItemsEntity[]> {
+  const command = `gh issue list --json "title,number,url,labels" --search "type:${issueType}" --limit 1000`;
```

Comment:
one bad side effect of this change is that a developer needs `gh` cli with proper token to build docs locally 🤔 

this needs at least documented!

Reply:
Yes, I will add.